core/src/main/resources/reference.conf

```diff
@@ -173,13 +173,20 @@ system {
     object = 128000
   }
 
-  # Rate at which Cromwell updates it's instrumentation gauge metrics (e.g: Number of workflows running, queued, etc..)
+  # Rate at which Cromwell updates its instrumentation gauge metrics (e.g: Number of workflows running, queued, etc..)
   instrumentation-rate = 5 seconds
   
   job-rate-control {
     jobs = 50
     per = 1 second
   }
+
+  workflow-heartbeats {
+    ttl: 10 minutes
+    heartbeats-per-ttl: 3
```

Comment:
OOC do you think there's a use case where someone would want to change this or is it only so that it's not hardcoded ?

Reply:
I was using it a lot during testing, though I could have accomplished something similar by tweaking `ttl` instead. 🙂 I was making knobs for everything else so I made one for this too. shrug